sample/Sample/ContentPopup/ContentPopupBGColorTest.xaml.cs

```diff
@@ -32,7 +32,7 @@ public ContentPopupBGColorTest()
             InitializeComponent();
         }
 
-        async void OnContentPopupTest1Clicked(object sender, EventArgs e)
+        void OnContentPopupTest1Clicked(object sender, EventArgs e)
```

Comment:
Probably better to name the test name more clear.

Reply:
I've fixed, thank you :)